### PR TITLE
JPS execution without environment

### PR DESCRIPTION
**Pull request by vlobzakov**
An ability added to create add-on w/o env: https://jelastic.team/browse/JE-22408

### CODE REVIEW

File: docs/creating-manifest/basic-configs.md [hunk 1]
```diff
@@ -890,4 +900,57 @@ onInstall:
...
+The JPS manifest can be executed without binding to any environment. The envName parameter is not mandatory for API anymore. 
+@@@
+```yaml
+type: install
+name: "Add-on without environment"
+
+description: |
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas euismod sapien eu risus venenatis, at faucibus dui cursus. Vestibulum a ipsum eget nibh aliquet vestibulum. Ut vestibulum ultrices sem quis ultricies. Pellentesque maximus nisl nec metus convallis egestas. Ut posuere leo quis semper tristique. In bibendum sem quis aliquet porttitor.
+
+startPage: https://jelastic.com/
```
Comment (SlavaKatiukha) [inline docs/creating-manifest/basic-configs.md]:
```suggestion
startPage: https://example.com/
```

File: docs/creating-manifest/basic-configs.md [hunk 2]
```diff
@@ -890,4 +900,57 @@ onInstall:
 }
 ```
 @@!
+
+## JPS execution without environment
```
Comment (SlavaKatiukha) [inline docs/creating-manifest/basic-configs.md]: Should be described as `type: install` without `nodes` parameter.
Types description should be changed also (see https://github.com/vlobzakov/docs/pull/18/files#diff-8579cb093488ca3b1e53aa719d602435R20)

File: docs/creating-manifest/basic-configs.md [hunk 3]
```diff
@@ -890,4 +900,57 @@ onInstall:
...
+ 
+The JPS manifest can be executed without binding to any environment. The envName parameter is not mandatory for API anymore.
```
Comment (SlavaKatiukha) [inline docs/creating-manifest/basic-configs.md]:
```suggestion
The JPS manifest can be executed without binding to any environment.  
```

File: docs/creating-manifest/basic-configs.md (hunk 2 above)
Comment (SlavaKatiukha) [reply docs/creating-manifest/basic-configs.md]: \+ move the description after `Environment Installation` section

File: docs/creating-manifest/basic-configs.md [hunk 4]
```diff
@@ -890,4 +900,57 @@ onInstall:
...
+
+onInstall:
+  - log: test
+```
+``` json
+{
+  "type": "install",
+  "name": "Add-on without environment",
+  "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas euismod sapien eu risus venenatis, at faucibus dui cursus. Vestibulum a ipsum eget nibh aliquet vestibulum. Ut vestibulum ultrices sem quis ultricies. Pellentesque maximus nisl nec metus convallis egestas. Ut posuere leo quis semper tristique. In bibendum sem quis aliquet porttitor.\n",
+  "startPage": "https://jelastic.com/",
```
Comment (SlavaKatiukha) [inline docs/creating-manifest/basic-configs.md]:
```suggestion
  "startPage": "https://example.com/",
```

**Review by SlavaKatiukha — CHANGES_REQUESTED** [review]

File: docs/creating-manifest/basic-configs.md (hunk 2 above)
Comment (vlobzakov) [reply docs/creating-manifest/basic-configs.md]: > + move the description after `Environment Installation` section

done

Comment (vlobzakov) [reply docs/creating-manifest/basic-configs.md]: > Should be described as `type: install` without `nodes` parameter.
> Types description should be changed also (see https://github.com/vlobzakov/docs/pull/18/files#diff-8579cb093488ca3b1e53aa719d602435R20)

Could you clarify what should be changed exactly?

**Review by SlavaKatiukha — APPROVED** [review]

File: docs/creating-manifest/basic-configs.md (hunk 2 above)
Comment (SlavaKatiukha) [reply docs/creating-manifest/basic-configs.md]: see: https://github.com/vlobzakov/docs/blame/master/docs/creating-manifest/basic-configs.md#L20

Comment (vlobzakov) [reply docs/creating-manifest/basic-configs.md]: что то я так и не понял суть коммента и в чем неточность. Можешь пояснить?

**Pull request merged**